coremltools/converters/mil/frontend/torch/ops.py

```diff
@@ -361,6 +361,31 @@ def constant(context, node):
     context.add(const, torch_name=name)
 
 
+
+@register_torch_op(override=True)
```

Comment:
Do we need the `override=True`?

Reply:
This was in the original implementation so I didn't change it. I also don't think we need this.

Reply:
Let's remove it.